### PR TITLE
[PR #6340/f6334e2 backport][3.9] Add an argument to override request body size (#6340)

### PR DESCRIPTION
**Pull request by anesabml**
(cherry picked from commit f6334e2b4881b9870edcaaa20e9282c4e9dc70f1)

## Checklist

- [ ] I think the code is well written
- [ ] Unit tests for the changes exist
- [ ] Documentation reflects the changes
- [ ] If you provide code modification, please add yourself to `CONTRIBUTORS.txt`
  * The format is &lt;Name&gt; &lt;Surname&gt;.
  * Please keep alphabetical order, the file is sorted by names.
- [ ] Add a new news fragment into the `CHANGES` folder
  * name it `<issue_id>.<type>` for example (588.bugfix)
  * if you don't have an `issue_id` change it to the pr id after creating the pr
  * ensure type is one of the following:
    * `.feature`: Signifying a new feature.
    * `.bugfix`: Signifying a bug fix.
    * `.doc`: Signifying a documentation improvement.
    * `.removal`: Signifying a deprecation or removal of public API.
    * `.misc`: A ticket has been closed, but it is not of interest to users.
  * Make sure to use full sentences with correct case and punctuation, for example: "Fix issue with non-ascii contents in doctest text files."

### CODE REVIEW

File: docs/web_reference.rst [hunk 1]
```diff
@@ -149,6 +149,23 @@ and :ref:`aiohttp-web-signals` handlers.
...
+
+      .. versionchanged:: 3.8
+
+         *Forwarded* and *X-Forwarded-Proto* are not used anymore.
+
+         Call ``.clone(client_max_size=new_size)`` for setting up the value
+         explicitly.
+
+      .. seealso:: :ref:`aiohttp-web-forwarded-support`
+
```
Comment (asvetlov) [inline docs/web_reference.rst]:
```suggestion
```
Wrong copy-paste

**Comment by anesabml** [comment]
If this PR #6343 is closed because it's a new feature, should I close this too?

**Comment by asvetlov** [comment]
Please wait.
aiohttp master is for 4.0, it still has incomplete work that must be finished before 4.0 final.
If we will have enough new features in 3.9 and 4.0 is still blocked -- a new release will be 3.9.0 from 3.9 branch.
In 3.9 we cannot make backward-incompatible changes but still can add new features.

I know, I should fix 3.9 before merging. The branch is broken most likely after the previous backport.

File: requirements/test.txt [hunk 2]
```diff
@@ -3,9 +3,10 @@ Brotli==1.0.9
 coverage==6.1.2
 cryptography==36.0.0; platform_machine!="i686" # no 32-bit wheels; no python 3.9 wheels yet
 freezegun==1.1.0
+# proxy.py==2.3.1
+https://github.com/abhinavsingh/proxy.py@master
```
Comment (webknjaz) [inline requirements/test.txt]: This branch is for stable releases. It is the same as v2.3.0: https://github.com/abhinavsingh/proxy.py/blob/master/requirements.txt. You may want to use `develop` instead. But then again, it's probably not something you need to change within this specific PR.
Plus I've been helping them revamp the whole packaging+CI/CD process entirely over the last month and there's a new version coming up soon. There's already an alpha release on PyPI created with the renewed packaging and fixed dependency pin if you _really_ want a bump:
```suggestion
proxy.py ~= 2.4.0a2
```
(`~=` is used to only allow pre-releases of this minor release; if you use `==` it'll implicitly use `--pre` in pip for all the versions of this package).

**Pull request merged**